cluster/internal/data.proto

```diff
@@ -23,7 +23,7 @@ message Tag {
 }
 
 message Point {
-	required string Name = 1;
+    required string Name = 1;
```

Comment:
ah, never mind then, I wasn't paying attention that it wasn't new. This is actually part of the writing across a cluster. No worries about this